package.json

```diff
@@ -30,17 +30,17 @@
   },
   "devDependencies": {
     "@babel/cli": "7.8.4",
-    "@babel/core": "7.9.0",
-    "@babel/preset-env": "7.9.5",
-    "@babel/preset-typescript": "7.9.0",
-    "@microsoft/api-extractor": "7.7.13",
-    "@rollup/plugin-alias": "3.1.0",
+    "@babel/core": "7.8.7",
+    "@babel/preset-env": "7.8.7",
+    "@babel/preset-typescript": "7.8.3",
+    "@microsoft/api-extractor": "7.7.8",
+    "@rollup/plugin-alias": "3.0.1",
     "@rollup/plugin-json": "4.0.2",
-    "@rollup/plugin-node-resolve": "7.1.3",
-    "@types/jest": "25.2.1",
-    "@types/node": "13.11.1",
-    "@typescript-eslint/eslint-plugin": "2.28.0",
-    "@typescript-eslint/parser": "2.28.0",
+    "@rollup/plugin-node-resolve": "7.1.1",
```

Comment:
I suspect either of the rollup plugins to have caused the regression. Can you try upgrading one at a time to see which is causing it?

Otherwise it could be babel too

Reply:
Honestly, it requires a little bit more time than I have right now. So I probably prefer do it later so I can make a release next Monday.